planetarypraxis/smartforests
# feat: add sort to more index pages and allow editing the published date for stories and radios

Adds the "First Published At" field to the settings page of stories and radio episodes.
Also adds the sort dropdown to the story and radio index pages.

## Motivation and Context
Required to sort and change order of these content types.

## How Can It Be Tested?
Edit a radio page in the admin to change the published date, view the radio index page, and check the sort is correct. Do the same for logbooks, stories, and logbook items.

## Types of changes


## Checklist:


- * I've checked the spec (e.g. Figma file) and documented any divergences.
- * My change requires a change to the documentation.
- * I've updated the documentation accordingly.
- * Replace unused checkboxes with bullet points.
